### PR TITLE
Chore/alexander/integrate create election api

### PR DESCRIPTION
**Pull request by xand6r**
This PR migrates the blockchain API from nodejs backend to react

### CODE REVIEW

File: src/components/landingPage/components/LandingPage.js [hunk 1]
```diff
@@ -67,50 +67,53 @@ const LandingPage = () => {
...
+                    </a>
+                    <a href={INSTAGRAM_PAGE}>
+                        <img className="--small" src={Instagram} alt="Instagram" />
+                    </a>
+                    <a href={TWITTER_PAGE}>
+                        <img className="--small" src={Twitter} alt="Twitter" />
+                    </a>
+                    <a href={YOUTUBE_PAGE}>
+                        <img className="--small" src={Youtube} alt="Youtube" />
+                    </a>
```
Comment (EngrEric) [inline src/components/landingPage/components/LandingPage.js:113]: don't do this, move this constants to array of objects and map over it.
also if this is a reusable footer, consider making it a component

Comment (xand6r) [reply src/components/landingPage/components/LandingPage.js:113]: i have corrected it in the next two PR's i am going to make

**Review by EngrEric — CHANGES_REQUESTED** [review]
Looks good, you just need to make minor changes. 
Why do you still have redux thunk and redux sagas

**Comment by xand6r** [comment]
> Looks good, you just need to make minor changes.
> Why do you still have redux thunk and redux sagas

do i? i have removed redux thunk though.

**Comment by xand6r** [comment]
> > Looks good, you just need to make minor changes.
> > Why do you still have redux thunk and redux sagas
> 
> do i? i have removed redux thunk though.

i will remove it from my package.json in upcoming PR's

**Review by usmanoa — APPROVED** [review]
Looks good

**Pull request merged**